Framework/MDAlgorithms/test/IntegratePeaksMD2Test.h

```diff
@@ -180,6 +180,119 @@ class IntegratePeaksMD2Test : public CxxTest::TestSuite {
     TS_ASSERT(algF.isExecuted());
   }
 
+  //-------------------------------------------------------------------------------
+  /** Setup simple algorithm*/
+  static void simpleRun(bool shouldPass, size_t numberValuesPeakRadius, size_t numberValuesBkgInnerRadius,
+                        size_t numberValuesBkgOuterRadius, bool cyl = false, bool ellip = false) {
+    createMDEW();
+    // --- Make a fake PeaksWorkspace ---
+    PeaksWorkspace_sptr peakWS(new PeaksWorkspace());
+    Instrument_sptr inst = ComponentCreationHelper::createTestInstrumentCylindrical(5);
+    peakWS->addPeak(Peak(inst, 1, 1.0, V3D(0., 0., 0.)));
+    TS_ASSERT_EQUALS(peakWS->getPeak(0).getIntensity(), 0.0);
+    AnalysisDataService::Instance().addOrReplace("IntegratePeaksMD2Test_peaks", peakWS);
+
+    std::vector<double> peakRadius(numberValuesPeakRadius, 1.0);
+    std::vector<double> bkgInnerRadius(numberValuesBkgInnerRadius, 1.1);
+    std::vector<double> bkgOuterRadius(numberValuesBkgOuterRadius, 2.0);
+
+    IntegratePeaksMD2 alg;
+    TS_ASSERT_THROWS_NOTHING(alg.initialize())
+    TS_ASSERT(alg.isInitialized())
+    TS_ASSERT_THROWS_NOTHING(alg.setPropertyValue("InputWorkspace", "IntegratePeaksMD2Test_MDEWS"));
+    TS_ASSERT_THROWS_NOTHING(alg.setProperty("PeakRadius", peakRadius));
+    TS_ASSERT_THROWS_NOTHING(alg.setPropertyValue("PeaksWorkspace", "IntegratePeaksMD2Test_peaks"));
+    TS_ASSERT_THROWS_NOTHING(alg.setPropertyValue("OutputWorkspace", "IntegratePeaksMD2Test_Output"));
+    TS_ASSERT_THROWS_NOTHING(alg.setProperty("BackgroundInnerRadius", bkgInnerRadius));
+    TS_ASSERT_THROWS_NOTHING(alg.setProperty("BackgroundOuterRadius", bkgOuterRadius));
+    TS_ASSERT_THROWS_NOTHING(alg.setProperty("Ellipsoid", ellip));
+    TS_ASSERT_THROWS_NOTHING(alg.setProperty("Cylinder", cyl));
+    std::cout << "Running simpleRun with inputs: shouldPass:" << shouldPass
+              << " numberValuesPeakRadius:" << numberValuesPeakRadius
+              << " numberValuesBkgInnerRadius:" << numberValuesBkgInnerRadius
+              << " numberValuesBkgOuterRadius:" << numberValuesBkgOuterRadius << " cyl:" << cyl << " ellip:" << ellip
+              << "\n";
```

Comment:
Use the our Mantid Logger instead of `std::cout` at information level instead (or alternatively debug if appropriate). This allows us to pipe outputs where we need them